packages/wrapt/test_wrapt.py

```diff
@@ -3,6 +3,7 @@
 
 @run_in_pyodide(packages=["wrapt"])
 def test_wrapt():
+    import inspect
```

Comment:
Is this needed?

Reply:
`inspect` is undefined otherwise. I believe this test never runs (see #2174).

Reply:
Oh yeah, I didn't notice it is used below. I forgot about this, thanks for the reminder.

Reply:
We should open an issue about it, as CI says that test_wapt.py passes, it probably doest run as expected.

Reply:
I wasn't able to figure out how to get this style of test to run in boost-histogram, so I just rewrote it. Pretty sure you need to manually force the class to run it's tests.

Reply:
Can't we just add `unittest.main()` to the end?

Reply:
The tests seem to run successfully and fail somewhere but then it results in a useless error message...

Reply:
Didn't seem to work for me with boost-histogram, but I might have had another issue, so should try it.